interpreter/text/parser.mly

```diff
@@ -566,42 +566,58 @@ func_body :
 
 /* Tables, Memories & Globals */
 
+table_ref :
```

Comment:
Rename to `table_use`, in analogy to the existing `type_use`.

Reply:
Done.